input/input.c

```diff
@@ -104,7 +106,10 @@ static int openInputDevice(lua_State *L) {
     } else {
         inputfds[fd] = open(inputdevice, O_RDONLY | O_NONBLOCK, 0);
         if (inputfds[fd] != -1) {
-            ioctl(inputfds[fd], EVIOCGRAB, 1);
+            if (ko_dont_grab_input == NULL) {
+              ioctl(inputfds[fd], EVIOCGRAB, 1);
```

Comment:
```suggestion
                ioctl(inputfds[fd], EVIOCGRAB, 1);
```

Reply:
(You didn't allow me to push, so please fix the spacing. ^_^)

Reply:
i don't understand the "allow you to push" comment - are you saying to add you to the repo as write?

Reply:
No, not at all. I thought you had disabled edits from maintainers, but it may be a GitHub issue with organizations.

https://github.community/t/how-can-we-enable-allow-edits-from-maintainers-by-default/2847

See https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork